analyzer/src/functTest/java/org/jboss/gm/analyzer/alignment/SimpleProjectWithCustomGroovyScriptFunctionalTest.java

```diff
@@ -90,6 +91,11 @@ public void verifyGroovyInjection() throws IOException, URISyntaxException, Mani
                 .hasOnlyOneElementSatisfying(l -> assertThat(l).contains("2.0.15.Final-redhat-00001"));
         assertTrue(lines.stream().anyMatch(s -> s.contains("CustomVersion( '1.0.1.redhat-00002', project )")));
         assertTrue(systemOutRule.getLog().contains("Attempting to read URL"));
+
+        assertFalse(FileUtils.readFileToString(new File(projectRoot, "settings.gradle"), Charset.defaultCharset())
+                .contains("x-pack"));
+        assertTrue(FileUtils.readFileToString(new File(projectRoot, "settings.gradle"), Charset.defaultCharset())
```

Comment:
I think these 2 statements can be made a little more compact

Reply:
Done :-)

Reply:
Love it!